### PR TITLE
cherry: add some documentation

### PR DESCRIPTION
**Pull request by aslilac**

### CODE REVIEW

File: cherry/src/App.tsx [hunk 1]
```diff
@@ -61,7 +61,7 @@ const init: AppState = {
 		{ instruction: { type: "putc_i", i: 110 }, editing: false },
 		{ instruction: { type: "putc_i", i: 100 }, editing: false },
 		{ instruction: { type: "putc_i", i: 33 }, editing: false },
-		{ instruction: {}, editing: true },
+		// { instruction: {}, editing: true },
```
Comment (jntrnr) [inline cherry/src/App.tsx:64]: this doesn't look docs-related?

Comment (aslilac) [reply cherry/src/App.tsx:64]: It's just a bit I had for testing, I meant to remove it before submitting 😅

Comment (sophiajt) [reply cherry/src/App.tsx:64]: Ahh, if it doesn't break anything, can you take this part of the PR out and land only the doc updates?

Comment (aslilac) [reply cherry/src/App.tsx:64]: Is it really that big of a deal?

Comment (sophiajt) [reply cherry/src/App.tsx:64]: Just trying to keep it fair for everyone

Comment (aslilac) [reply cherry/src/App.tsx:64]: If you don't wanna merge it then whatever 😅 I was just here to have fun, I don't really care enough to go back an un-fix something

**Pull request closed without merging**